src/policy/policy.cpp

```diff
@@ -266,13 +273,18 @@ bool IsWitnessStandard(const CTransaction& tx, const CCoinsViewCache& mapInputs)
 
         // Check policy limits for Taproot spends:
         // - MAX_STANDARD_TAPSCRIPT_STACK_ITEM_SIZE limit for stack item size
-        // - No annexes
+        // - Limited annex format
         if (witnessversion == 1 && witnessprogram.size() == WITNESS_V1_TAPROOT_SIZE && !p2sh) {
             // Taproot spend (non-P2SH-wrapped, version 1, witness program size 32; see BIP 341)
             Span stack{tx.vin[i].scriptWitness.stack};
             if (stack.size() >= 2 && !stack.back().empty() && stack.back()[0] == ANNEX_TAG) {
-                // Annexes are nonstandard as long as no semantics are defined for them.
-                return false;
+                const auto& annex = stack.back();
+
+                if (annex.size() >= annex_bytes_left) return false;
+                annex_bytes_left -= annex.size() - 1; // bip141 witness stack element size included
```

Comment:
`-= size() + 1` ?

If you do `+1` then the maximum annex data is 129 witness bytes, if you do `-1` then you could could have 129 "0x50 0x00"  entries for a total of 258 witness bytes?

Reply:
typo.